client/src/components/Grid.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
```

Comment:
It's a small thing, but no need to import { Component } is you're only going to use "React.Component" importing React is enough.